programs/monaco_protocol/src/state/market_liquidities.rs

```diff
@@ -0,0 +1,414 @@
+use std::cmp::Ordering;
+use std::string::ToString;
+
+use anchor_lang::prelude::*;
+
+use crate::error::CoreError;
+use crate::state::type_size::*;
+
+#[account]
+pub struct MarketLiquidities {
+    pub market: Pubkey,
+    pub liquidities_for: Vec<MarketOutcomePriceLiquidity>,
+    pub liquidities_against: Vec<MarketOutcomePriceLiquidity>,
+}
+
+impl MarketLiquidities {
+    pub const SIZE: usize = DISCRIMINATOR_SIZE
+        + PUB_KEY_SIZE // market
+        + vec_size(18_usize, 0_usize) // liquidities_for
+        + vec_size(18_usize, 0_usize); // liquidities_against
```

Comment:
NP `18` is a bit of a magic number here. Usually I think we would implement size_of for the struct in question, and use that here but it's fine.

Reply:
Ok. Let's do it now.